check_strategies.py

```
@@ -0,0 +1,61 @@
+
+# Login endpoint
+url = "https://cryptouniverse.onrender.com/api/v1/auth/login"
+
+# Admin credentials
+payload = {
+    "email": "admin@cryptouniverse.com",
+    "password": "AdminPass123!"
+}
+
+# Make the login request
+response = requests.post(url, json=payload)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)